### ISSUE TITLE
Synching forever

### ISSUE THREAD

**Issue by JasonThomasData** [openaustralia/morph#1028]
I run Ubuntu 14.04 and use Chrome
I've been looking at this screen for ten minutes (yes haha I have no life).

![image](https://cloud.githubusercontent.com/assets/9067490/17013824/2f8ca45e-4f64-11e6-8aed-cddc6d5ea31a.png)

If I go to my account I can see the scraper is integrated and all, but that information's not being fed back to me, it seems.

**Comment by henare** [comment]
Hmm, I just got this at the same point. Linux Mint/Firefox.

![morph io planningalerts-scrapers-wyndham_city_council - mozilla firefox_001](https://cloud.githubusercontent.com/assets/48945/17014372/a031ffa8-4f67-11e6-9131-beee59aa6254.png)

Could this be related to the upgrades today @mlandauer?

**Comment by mlandauer** [comment]
This looks to have magically sorted itself out. I checked both and they seem fine now. So, closing this.

**Comment by henare** [comment]
The issue for both was that they silently completed in the background but to the end user this is all that was displayed.

I won't reopen until we can reliably reproduce, I guess.

**Comment by mlandauer** [comment]
So, just to make sure I understand, the issue was that the page didn't update in realtime as the synching was going on but then when the page was reloaded showed that everything worked?

**Comment by henare** [comment]
Correct. The page got stuck at that step for both of us. If we just hit refresh it would appear already magically done.

**Comment by mlandauer** [comment]
I can't reproduce the problem on my local development machine.

**Comment by henare** [comment]
I didn't see it when smoke testing locally yesterday either.

I can reproduce in production though. Here's the waterfall:

![selection_001](https://cloud.githubusercontent.com/assets/48945/17043079/a16d62ec-4ff6-11e6-85f1-bd93f10ae016.png)

As you can see, that last Faye call just sits there for ages then it seems to timeout. Here's the response:

     /**/__jsonp3__([{"id":"5","clientId":"[redacted]","channel":"/meta/connect","successful":true,"advice":{"reconnect":"retry","interval":0,"timeout":45000}}]);

**Comment by benrfairless** [comment]
Thanks for raising this. While reviewing the morph.io backlog, it looks like the work described here was largely addressed some time ago, so we are closing it as resolved. Some smaller parts may not have been fully completed.

If you find this is still an issue, please reopen it or open a fresh issue with current detail and we will take another look.

This triage was carried out with AI assistance (Claude Opus 4.8 via Claude Code) and reviewed by the OpenAustralia Foundation team.
